### DISCUSSION TITLE
How do I run a local copy of this plugin for contributing

### DISCUSSION THREAD

**Discussion by ayhid** [VirtusLab-Open-Source/strapi-plugin-comments#167] [Q&A]
Hello,
I want to contribute to this plugin and play around on development mode.

Do I have to install strapi separately and add the plugin ?

**Comment by cyp3rius** [comment]
Hello @ayhid,

The things you need to do are:

- run strapi as separated project
- clone our repository
- do the soft link between `your_strapi_directory/src/plugins` and `strapi-plugin-comments/build`
- run plugin in development mode as well as strapi

That's the way we're working locally with multiple our plugins.
